invenio_access/models.py

```diff
@@ -134,3 +136,50 @@ class ActionRoles(ActionNeedMixin, db.Model):
     def need(self):
         """Return RoleNeed instance."""
         return RoleNeed(self.role.name)
+
+
+def removed_or_inserted_action(mapper, connection, target):
+    """Remove the action from cache when an item is inserted or deleted."""
+    del mapper, connection
```

Comment:
Why do you need to call `del`? The reference counters get automatically updated when function returns.